packages/react-generator/src/templates/react-template/template-files/src/lowcodeConfig/dataSource.js

```
@@ -0,0 +1,102 @@
+  const shouldFetch = config.shouldFetch?.value ? createFn(config.shouldFetch.value) : () => true
+  const willFetch = config.willFetch?.value ? createFn(config.willFetch.value) : (options) => options
+
+  const dataHandler = (res) => {
+    const data = config.dataHandler?.value ? createFn(config.dataHandler.value)(res) : res
+    dataSource.status = 'loaded'
+    dataSource.data = data
+    return data
+  }
+
+  const errorHandler = (error) => {
+    config.errorHandler?.value && createFn(config.errorHandler.value)(error)
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)